lib/src/Base/Geom/IntervalMesher.cxx

```diff
@@ -192,6 +192,10 @@ Mesh IntervalMesher::build(const Interval & interval,
   } // dimension == 2
   if (dimension == 3)
   {
+    // FIXME: https://github.com/openturns/openturns/issues/1670
+    if (diamond)
+      throw NotYetImplementedException(HERE) << "In IntervalMesher::build with 3-d/diamond=true";
```

Comment:
I started a branch to fix this one, but it is not so easy...

Reply:
I dont think its urgent